src/main/java/com/celements/web/service/CelementsWebScriptService.java

```diff
@@ -1077,25 +1077,26 @@ public String getVersion(String systemComponentName) {
     return this.versionMap.get(systemComponentName);
   }
 
-  public MetaTagApi getCharsetMetaTag(String charsetStr) {
+  public MetaTag getCharsetMetaTag(String charsetStr) {
```

Comment:
please move MetaTag ScriptService calls to own ScriptService class e.g. MetaTagScriptService ("metatag").